benchmark/single-source/Hash.swift

```diff
@@ -113,18 +113,16 @@ class Hash {
   /// \brief Left-rotate \p x by \p c.
   final
   func rol(_ x: UInt32, _ c: UInt32) -> UInt32 {
-    // TODO: use the &>> operator.
-    let a = UInt32(truncatingIfNeeded: Int64(x) << Int64(c))
-    let b = UInt32(truncatingIfNeeded: Int64(x) >> (32 - Int64(c)))
+    let a = UInt32(truncatingIfNeeded: Int64(x) &<< Int64(c))
+    let b = UInt32(truncatingIfNeeded: Int64(x) &>> (32 - Int64(c)))
```

Comment:
These still aren't as simple as they should be. This can be:
~~~~
return x &<< c | x &>> (32 &- c)
~~~~
(or even `0 &- c`, thanks to how masking works.)